_posts/2018-06-25-refactoring-with-design-patterns.markdown

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: "Refactoring: clean your ruby code with design patterns"
+date: 2018-06-25 12:00:00
+categories: ["code-refactor"]
```

Comment:
@etagwerker we had to remove it since there seems to be a bug in the categories plugin. We already have a `design` category, and it overwrites that category for some reason and removes all articles under the `design` category. So I asked @cleicar to choose an alternative to work around that bug.

Reply:
@mauro-oto Ah, got it, thanks!